main.tf

```diff
@@ -25,7 +25,7 @@ module "codepipeline" {
 }
 
 module "codebuild" {
-  source = "git::https://github.com/launchbynttdata/tf-aws-module_collection-codebuild.git?ref=1.0.0"
+  source = "git::https://github.com/launchbynttdata/tf-aws-module_collection-codebuild.git?ref=1.0.2"
```

Comment:
Can we swap this over to the registry URLs? This is one of the repos that my automation couldn't process, I just haven't had time to PR the changes.

Reply:
Done!!